Update deployment user permissions

solumath opened the pull request:
## PR type
<!-- check all applicable -->
- [x] Bug Fix

## Description
With the new deployment, there is a problem with files having permissions conflicts when git pulling. This should introduce dynamic permissions enabling us to be able to manually use /git pull but also have the GitHub action workflow work as expected.
Updated readme about installation steps.

## Related Issue(s)
None

## After checks
<!-- check all applicable -->
- [x] PR was tested
- [ ] Major change (packages, libraries, etc.)

## Post deployment
<!-- [Optional] Are there any post-deployment tasks we need to perform? -->
Breaking change. After merging we need to change ownerships of all files to local user, EXCEPT POSTGRESL.SQL!
Users also need to generate an .env file.

## UI changes
<!-- [Optional] If there are UI changes, please paste screenshot(s) -->

1ukastesar commented on .gitignore:
```diff
@@ -5,6 +5,7 @@ __pycache__
 .idea
 venv
 .venv
+.env
```
So are we using `.env`, `.venv` or `venv` (lines above)? We should decide which one to use and put here only one of the variants.

I would vote for `env` as it is default in many guides and I use this variant in a lot of other projects, but it's just my opinion.

peterdragun replied:
I don't think we should even have those in the repository, this is more specific to user setup and should be part of global gitignore for every user.

solumath replied:
1. For me it doesn't matter that much if there are all versions of popular `venv, .venv, env, .env` since everyone uses different name for it.

2. I would agree with having global setup, but a lot of newcomers contribute to this project and it's better like this in my opinion. We don't have to tell them all the time about setting it up.

1ukastesar commented on logs/README.md:
Isn't it better to use `.gitignore` for the purpose of storing an empty folder in Git? Empty `README.md` is a little confusing IMO.

solumath replied:
except this is not empty readme, this is only change of permissions for readme

solumath commented:
As for update
- I've reorganized gitignore and removed unused names.
- Updated readmes for folders that are used only on runtime. Git doesn't support empty dirs and in my opinion readmes are better since they explain why this directory exists.
- Update compose to be able to use `docker compose up --build` for local development

peterdragun commented on .gitignore:
```diff
@@ -1,17 +1,27 @@
+merlin-latest
 service_logs.txt
-emojis.zip
+config/config.toml
```
you are missing backup config

solumath replied:
as discussed we no longer need manual backup as we have cronjob on server for that

peterdragun commented on init.sh:
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DEFAULT_UID=$(id -u)
+DEFAULT_GID=$(id -g)
+
+# Create logs folder with permissions
+echo "Updating logs folder permissions"
+mkdir -p logs
+sudo chmod -R 777 logs
+sudo chown -R $DEFAULT_UID:$DEFAULT_GID logs/
```
Is this still needed?

solumath replied:
Yes, this will be used for creating logs folder with correct permissions. This is needed for the setup of the bot. We can later add more commands to init or remove it the file as whole if we find better solution with docker permissions.
